titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/eviction/EvictionDataSnapshot.java

```diff
@@ -0,0 +1,125 @@
+package com.netflix.titus.runtime.connector.eviction;
+
+import java.util.Map;
+import java.util.Optional;
+
+import com.netflix.titus.api.eviction.model.EvictionQuota;
+import com.netflix.titus.api.eviction.model.SystemDisruptionBudget;
+import com.netflix.titus.api.model.Reference.TierReference;
+import com.netflix.titus.api.model.Tier;
+
+import static com.netflix.titus.common.util.CollectionsExt.copyAndAdd;
+
+public class EvictionDataSnapshot {
+
+    private final SystemDisruptionBudget globalDisruptionBudget;
+    private final Map<Tier, SystemDisruptionBudget> tierSystemDisruptionBudges;
```

Comment:
tierSystemDisruptionBudges -> tierSystemDisruptionBudget

Reply:
Fixed here and in the other places.